#! /bin/sh /usr/share/dpatch/dpatch-run
## 001readme.dpatch by  <andrea.de.iacovo@gmail.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fixing reame file

@DPATCH@

diff -Nru wordpress/readme.html wordpress-2.5.0/readme.html
--- wordpress/readme.html	2008-02-29 18:09:44.000000000 +0100
+++ wordpress-2.5.0/readme.html	2008-04-14 14:08:58.000000000 +0200
@@ -85,7 +85,7 @@
 <p>WordPress has no multi-million dollar marketing campaign or celebrity sponsors, but we do have something even better&#8212;you. If you enjoy WordPress please consider telling a friend, setting it up for someone less knowledgable than yourself, or writing the author of a media article that overlooks us.</p>
 
 <h1>Copyright</h1>
-<p>WordPress is released under the <abbr title="GNU Public License">GPL</abbr> (see <a href="license.txt">license.txt</a>).</p>
+<p>WordPress is released under the <abbr title="GNU Public License">GPL</abbr> (see <a href="http://www.gnu.org/copyleft/gpl.html">license.txt</a>).</p>
 
 </body>
 </html>
